tests/unit/api/test_bootstrap.py

```diff
@@ -76,16 +76,6 @@ def test_get_bootstrap_incorrect_scope_token(
             "Authorization": f"Bearer {token.json()['access_token']}",
         }
         url = "/api/v1/bootstrap/"
-        mocked_check_metadata = pretend.call_recorder(lambda: False)
```

Comment:
We still check the bootstrap state.
Why can we remove this part of the code?

Reply:
It will fail on the API level (fastapi) because the token is invalid.
It is not processed.